src/Components/CreateDog/CreateDog.js

```diff
@@ -0,0 +1,85 @@
+import { useEffect, useState } from "react";
+import "./CreateDog.styles.scss";
+import axios from "axios";
+
+function DogCardInfo() {
+  const [dogName, setDogName] = useState([]);
+  const [dogInfo, setDogInfo] = useState([]);
+
+  useEffect(() => {
+    RandomDogInfo();
+    RandomName();
+    modifiedObject();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [setDogName, setDogInfo]);
+
+  //This function is responsible for getting image, breed Name, size into the cards
+  //However it also saves the rest of the unused data.
+  const RandomDogInfo = async () => {
+    const key = "9/8P1dgeiaMQRHYSg9SgkA==olxKKsYDWHZlEAYU";
+    await axios
+      .get(`https://api.api-ninjas.com/v1/dogs?trainability=5`, {
+        headers: { "X-Api-Key": key },
+      })
+      .then((results) => {
+        setDogInfo(results.data);
+      });
+  };
+
+  //Requests a random name, its currently on a loop for 12 times
+  const RandomName = async () => {
+    await axios
+      .get("https://randomuser.me/api/?results=20&nat=us&inc=name")
+      .then((results) => {
+        setDogName(results.data.results.map((a) => a.name.first));
+      });
+  };
+
+  const modifiedObject = () => {
+    const list = dogInfo;
+    const girl = dogName;
```

Comment:
I dont understand why dogName equals girl

Reply:
This has been changed and has already been resolved, I did notice the girl variable and I removed it. The other thing I noticed is that I didn't need those two variables anymore since I can declare them into the function.